code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -11,13 +11,16 @@
 	smoothing_flags = NONE
 	var/growth_time = 120 SECONDS
 
-/obj/structure/alien/resin/flower_bud_enemy/New()
+/obj/structure/alien/resin/flower_bud_enemy/Initialize(mapload)
 	..()
 	var/list/anchors = list()
-	anchors += locate(x-2,y+2,z)
-	anchors += locate(x+2,y+2,z)
-	anchors += locate(x-2,y-2,z)
-	anchors += locate(x+2,y-2,z)
+	anchors += locate(x - 2, y + 2, z)
+	anchors += locate(x + 2, y + 2, z)
+	anchors += locate(x - 2, y - 2, z)
+	anchors += locate(x + 2, y - 2, z)
+
+	for(var/obj/structure/alien/weeds/W in get_turf(src))
```

Comment:
Hmmm

Do you think you could add a var to the parent type or something to prevent these from spawning in the first place? It's a little gross that we spawn it and then immediately delete it.

Reply:
That is a neater solution, will do once I'm home